docs/griptape-cloud/tools/create-tool.md

```diff
@@ -0,0 +1,17 @@
+# Tools
+
+Tools can be used to execute arbitrary actions in a request/response format. They can be given to an LLM so the LLM can take appropriate action based on the input, Tool description, and the description of each Tool activity.
```

Comment:
"Tool description, and the description of each Tool activity" is a confusing read.